### ISSUE TITLE
Repeating 'f' in iSearch mode within magit-log

### ISSUE THREAD

**Issue by esiegel** [syl20bnr/spacemacs#797]
I'm having the problem where searching for text within **magit-log** and pressing the character **f** causes the character to repeat continuously until I press **C-g**.  This problem occurs both in the terminal and in the GUI.

It seems that the **f** character is mapped to the command **evil-escape-isearch**, which I can't seem to find anywhere.

Strangely enough, the mapping to evil-escape-isearch exists when using isearch anywhere, but the repeating f's are exclusive to Magit.

If I could find the escape command I could potentially diagnose further how evil and magit are conflicting.

**EDIT:**
Spacemacs v.0.65.1

```emacs-lisp
(setq-default
   ; Disable auto-complete and dependencies so that we can use company for completion
   dotspacemacs-excluded-packages '(auto-complete
                                    ac-ispell
                                    auto-complete-clang
                                    enslime
                                    edts
                                    tern-auto-complete)

    ; List of contribution to load."
    dotspacemacs-configuration-layers '(colors
                                        company-mode
                                        fasd
                                        git
                                        go
                                        haskell
                                        html
                                        javascript
                                        markdown
                                        python
                                        restclient
                                        ruby
                                        scala
                                        themes-megapack)
)

```

**Comment by syl20bnr** [comment]
I just reproduced it within a paradox buffer. Thanks for the detailed report.

**Comment by syl20bnr** [comment]
Fixed in evil-escape version 2.14 (will be available in MELPA in a couple of hours).

**Comment by esiegel** [comment]
Awesome!  Thanks for the quick response. I'll place the link to the fix below for reference.

https://github.com/syl20bnr/evil-escape/commit/67843e9dd4518fba8aa8a26170653a94a7a1c27d
